src/main/java/com/hedera/exchange/database/ExchangeRateAWSRD.java

```diff
@@ -81,26 +81,6 @@
 
 	private static final String LATEST_ADDRESSBOOK_QUERY = "SELECT e1.expirationTime, e1.addressBook FROM address_book AS e1 INNER JOIN (SELECT MAX(expirationTime) expirationTime FROM address_book WHERE networkName = ?) AS e2 ON e1.expirationTime = e2.expirationTime LIMIT 1";
```

Comment:
Please open a ticket and assign it to me so I can check why the query fails if it is multiline

Reply:
https://github.com/hashgraph/hedera-exchange-rate-tool/issues/132